# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-ppc-abiversion.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-103.0.1/security/nss/lib/freebl/chacha20-ppc64le.S.vanilla	2022-08-12 15:36:44.125085032 +0200
+++ firefox-103.0.1/security/nss/lib/freebl/chacha20-ppc64le.S	2022-08-12 15:36:49.111084833 +0200
@@ -142,7 +142,7 @@
 #define vs62	62
 #define vs63	63
 
-.abiversion 2
+.abiversion 1
 .section ".data"
 .align 5
 lblock:	.skip 256
--- firefox-103.0.1/security/nss/lib/freebl/sha512-p8.s.vanilla	2022-08-12 15:35:58.104086867 +0200
+++ firefox-103.0.1/security/nss/lib/freebl/sha512-p8.s	2022-08-12 15:36:13.450086255 +0200
@@ -3,7 +3,7 @@
 # See the full LICENSE under scripts/.
 
 .machine	"any"
-.abiversion	2
+.abiversion	1
 .text
 
 .globl	sha512_block_p8
